apps/data/admin.py

```diff
@@ -1 +1,5 @@
-# Register your models here.
+from django.contrib import admin
+from .models import Repository, Entry
```

Comment:
If you agree with my first remark, apply the same idiom here.

Reply:
Done